src/elements/VolumesTable/VolumesTableView.js

```diff
@@ -133,35 +150,25 @@ const VolumesTableView = ({
                 className="input-row__item input-row__item_edit"
                 floatingLabel
                 type="text"
+                tip="A mount path for referencing the data from the function"
               />
             </div>
             <div
-              className={`input-row-wrapper
+              className={`input-row-wrapper no-border_top
                   ${newVolume.type === 'V3IO' && 'no-border'}`}
             >
-              <Select
-                onClick={type => {
-                  setNewVolume(state => ({
-                    ...state,
-                    type: find(selectTypeOptions.volumeType, ['id', type]).id
-                  }))
-                }}
-                options={selectTypeOptions.volumeType}
-                label={newVolume.type.length ? newVolume.type : 'Type'}
-              />
               <Input
                 onChange={typeName =>
                   setNewVolume(state => ({ ...state, typeName }))
                 }
-                label={volumeTypeNameLabel}
-                className="input-row__item input-row__item_edit"
+                label={volumeTypeNameLabel.label}
+                className="input-row__item"
                 disabled={newVolume.type.length === 0}
                 floatingLabel
                 type="text"
+                tip={volumeTypeNameLabel.tip ?? volumeTypeNameLabel.tip}
```

Comment:
Isn't this equivalent to:
```suggestion
                tip={volumeTypeNameLabel.tip}
```
?